docs/src/RunningOnGPUs.md

```diff
@@ -45,24 +45,25 @@ ITensor currently provides
 package extensions for the following GPU backends:
 
 * [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl) (NVIDIA GPUs)
+* [cuTENSOR.jl] (https://github.com/JuliaGPU/CUDA.jl/tree/master/lib/cutensor) (Submodule of `CUDA.jl`) 
 * [Metal.jl](https://github.com/JuliaGPU/Metal.jl) (Apple GPUs)
 * [AMDGPU.jl](https://github.com/JuliaGPU/AMDGPU.jl) (AMD GPUs)
 
 Our goal is to support all GPU backends which are supported by the [JuliaGPU organization](https://juliagpu.org).
 
+Notice that `cuTENSOR.jl` is based on `CUDA.jl` and provides new functionality for accelerated binary tensor contractions. If the `cuTENSOR` library is loaded then ITensors with `CuArray` data are contracted using `cuTENSOR` and if the `cuTENSOR` library is not loaded then binary tensor contractions are mapped to a matrix multiplication and performed using `cuBLAS`. It is advised as of now to not use `cuTENSOR` with `BlockSparse`.
```

Comment:
```suggestion
Notice that `cuTENSOR.jl` is an extension of `CUDA.jl` that provides new functionality for accelerated binary tensor contractions. If the `cuTENSOR.jl` library is loaded then ITensors with `CuArray` data are contracted using `cuTENSOR` and if the `cuTENSOR.jl` library is not loaded but `CUDA.jl` is loaded then binary tensor contractions are mapped to a matrix multiplication and performed using `cuBLAS`.
```
I don't think we have have to say "It is advised as of now to not use `cuTENSOR` with `BlockSparse`." since we already have a warning about block sparse operations on GPU below.